apps/go-mod-validator/src/deps.ts

```diff
@@ -0,0 +1,160 @@
+import { execSync } from "child_process";
+import { dirname } from "path";
+import * as core from "@actions/core";
+import * as glob from "@actions/glob";
+
+/**
+ * Taken from go help list for the command `go list -json -m all`
+ *
+ *  type Module struct {
+ *        Path       string        // module path
+ *        Query      string        // version query corresponding to this version
+ *        Version    string        // module version
+ *        Versions   []string      // available module versions
+ *        Replace    *Module       // replaced by this module
```

Comment:
I wonder if we need to take into account this directive when figuring out which versions are in use.

Reply:
We do, see goModsToGoModules function